src/components/AppTestimonialCard.vue

```diff
@@ -0,0 +1,46 @@
+<template>
+  <div class="card testimonial-card">
+    <img :src="this.imageSrc" class="card-img-top" alt="...">
+    <div class="card-body">
+      <h5 class="card-title text-left testimonial-card-heading">{{this.title}}</h5>
+      <p class="card-text text-left testimonial-card-body">{{this.body}}</p>
+    </div>
+  </div>
+</template>
+
+<script>
+export default {
+  name: "TestimonialCard",
+  props : {
+    imageSrc : String,
+    title: String,
+    body: String
+  }
+}
+</script>
+
+<style scoped>
+
+/*Testimonial Card*/
+.testimonial-card{
+  background-color: rgba(224, 221, 221, 0.88);
```

Comment:
Can we either define this (bg color) and the corresponding dark color scheme shade in the base.css or use one of the available colors from it here. 
The idea is that all the colours in the palette will be defined in `base.css` and anywhere else we will be using just the color variables, so if we completely change the color palette for the website, we just need to change the colors in that file only, without worrying about individual components.

Reply:
Done